mlflow/store/model_registry/databricks_workspace_model_registry_rest_store.py

```diff
@@ -0,0 +1,31 @@
+from mlflow.store.model_registry.rest_store import RestStore
+from mlflow.exceptions import MlflowException
+
+
+def _raise_unsupported_method(method, message=None):
+    messages = [
+        f"Method '{method}' is unsupported for models in the Workspace Model Registry. "
+        f"Upgrade to Models in Unity Catalog to access the latest features. You can configure "
+        f"the MLflow Python client to access models in Unity Catalog by running "
+        f"mlflow.set_registry_uri('databricks-uc') before accessing models.",
+    ]
+    if message is not None:
+        messages.append(message)
+    messages.append("See the user guide for more information")
+    raise MlflowException(" ".join(messages))
+
+
+class DatabricksWorkspaceModelRegistryRestStore(RestStore):
+    def set_registered_model_alias(self, name, alias, version):
+        _raise_unsupported_method(method="set_registered_model_alias")
+
+    def delete_registered_model_alias(self, name, alias):
+        _raise_unsupported_method(method="delete_registered_model_alias")
+
+    def get_model_version_by_alias(self, name, alias):
+        _raise_unsupported_method(
+            method="get_model_version_by_alias",
+            message="If attempting to load a model version by alias via a URI of the form "
+            "'models:/model_name@alias_name', configure the MLflow client to target Unity Catalog "
+            "and try again.",
```

Comment:
To `configure the MLflow client to target Unity Catalog`, what do we need to do?

Reply:
Oh I see, we need to run `mlflow.set_registry_uri('databricks-uc')`